lib/SILOptimizer/ARC/ARCRegionState.cpp

```diff
@@ -282,6 +287,18 @@ bool ARCRegionState::processLoopBottomUp(
       OtherState->second.checkAndResetKnownSafety(
           I, OtherState->first, checkIfRefCountInstIsMatched, RCIA, AA);
     }
+#ifndef NDEBUG
+    // Verify updateForDifferentLoopInst is conservative enough that the flow
+    // sensitive native of the loop summarized instructions does not matter.
+    if (verifyARCLoopSummary) {
```

Comment:
Is this verification inside the loop over SummarizedInterestingInsts? Shouldn't it be outside? Otherwise it's unnecessarily quadratic

Reply:
Thanks. I fixed it.